tools/accuracy_checker/accuracy_checker/preprocessor/audio_preprocessing.py

```diff
@@ -15,6 +15,7 @@
 """
 
 import numpy as np
+import scipy.signal as dsp
```

Comment:
please wrap to unsupported package, scipy is trouble maker for some platforms, we avoid to have it as explicit dependency if user do not want to use models where it is really required

Reply:
Fixed